### DISCUSSION TITLE
Show Windows Build Info on the Desktop Issue

### DISCUSSION THREAD

**Discussion by mwschramm77** [valinet/ExplorerPatcher#2376] [General]
Hi,

When I choose to 'Show Windows Build Info on Desktop', it works fine on my primary monitor (Windows Monitor 1), but the text is hidden behind the task bars at the bottom of Monitors 2 and 3.

![Monitor_2_Show_Windows_Build_Info](https://github.com/valinet/ExplorerPatcher/assets/57922114/a92a2504-b51c-48a5-8a4c-7ad32079f915)

**Comment by pyrates999** [comment]
Please do not post the same thing in issues and discussion.  Please close this one then in issues: https://github.com/valinet/ExplorerPatcher/issues/2375

**Comment by pyrates999** [comment]
Please post windows version and build. You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Comment by mwschramm77** [comment]
Closed the other issue:

Device name	PS473
Processor	13th Gen Intel(R) Core(TM) i9-13900KF   3.00 GHz
Installed RAM	128 GB (128 GB usable)
Device ID	3CF84C8E-8D27-4C64-BDD9-1F3BB899A35C
Product ID	00355-61080-77178-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Edition	Windows 11 Pro
Version	22H2
Installed on	‎9/‎13/‎2023
OS build	22621.2506
Experience	Windows Feature Experience Pack 1000.22677.1000.0

Running ExplorerPatcher 22621.2428.59.1

**Comment by pyrates999** [comment]
You got a preview update installed.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Comment by mwschramm77** [comment]
Thank you.  I just did a reboot and the issue 'cleared up'  OS Build info showing on all three monitors as expected.

BTW, I really like the work you are doing.  Thank  you so much for Explorer Patcher!
